src/api/operator/utils.ts

```diff
@@ -68,6 +68,28 @@ export function getOrderFilledAmount(order: RawOrder): { amount: BigNumber; perc
   return { amount: executedAmount, percentage: executedAmount.div(totalAmount) }
 }
 
+export function getSurplus(
+  inputAmount: BigNumber | string,
+  executedAmount: BigNumber,
```

Comment:
it's a bit strange you allow string for the input and not the executedAmount. Not biggy, I think u did it for not handling the case for executedAmount  and while for inputAmount was coming for free

Reply:
Mostly because executed amount will always be a BigNumber (from the sources I have so far).
But you are right, will make both inputs the same type